examples/Azure_IoT_Central_ESP32_AzureIoTKit/AzureIoT.h

```diff
@@ -0,0 +1,809 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
same here :)

Reply:
You are correct. I replicated all the changed between the 2 samples. These files are common between the samples.